raisinbread/instruments/bmi.linst

```diff
@@ -1,4 +1,3 @@
-table{@}bmi
```

Comment:
could we maybe add a .meta file to this PR if we're removing this line from the raisinbread example?

Reply:
I thought I had but it seems like i didnt submit it. ma bad